scripts/get-prs.py

```diff
@@ -58,7 +58,9 @@ def extract_gitref(s):
     ]
 else:
     for c in commits:
-        prs.update(c.get_pulls())
+        if len(c.parents() == 1):
```

Comment:
Looks like it should be:

```suggestion
        if len(c.parents == 1):
```

```
Traceback (most recent call last):
  File "/Users/minrk/dev/jpy/hub/mybinder.org-deploy/scripts/get-prs.py", line 61, in <module>
    if len(c.parents() == 1):
TypeError: 'list' object is not callable
```

Reply:
It shouldn't matter. The script lists all commits between the two SHAs, and finds the PR associated with each of those commits. There should always be a non-merge commit alongside the merge commit, so the PR should still be included